.circleci/config.yml

```diff
@@ -32,18 +32,16 @@ jobs:
       GOMAXPROCS: "3"
     steps:
       - checkout
-      - run: make init
-      - run: ./ci/install-lint
-      - run: ./ci/lint
-          
+      - run: make lint
+
   test:
     executor: builder
     resource_class: medium+
     steps:
       - checkout
       - run: make init
-      - run: ./ci/install-envoy
-      - run: ./ci/test
+      - run: go run cmd/getenvoy/main.go fetch standard:1.11.0
```

Comment:
this deserves a comment, if someone can remember why this was done.

Reply:
would it be possible to change this to `standard:1.17.1` which is the current latest?

Reply:
yeah let's